src/app/pre-sale/components/pre-sale-token-card/index.tsx

```diff
@@ -77,7 +77,9 @@ export function PreSaleTokenCard({ token, onClick }: Props) {
       </div>
       <button
         className='text-ic-white bg-ic-blue-600 w-full rounded-lg py-2.5 font-bold disabled:cursor-not-allowed disabled:bg-[#CFD9D9]'
-        disabled={token.status !== PreSaleStatus.ACTIVE}
+        disabled={
+          token.status !== PreSaleStatus.ACTIVE || onClick === undefined
```

Comment:
thinking through this - when would `onClick` be undefined with an `ACTIVE` status?

Reply:
@0xonramp when it's shown in the pre-sale popup cause it doesn't make sense to show an active `join pre-sale` button here. But we still would want to keep the PreSaleStatus `ACTIVE`.